masterlist.yaml

```
@@ -3335,6 +3335,31 @@ plugins:
       - crc: 0xE5BCCA66
         util: 'TES4Edit v4.0.4'
 
+  - name: '(Knights-OpenCrusaderHelm|EVE_Knights-OpenCrusaderHelm)\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/49319/'
+        name: 'Knights of the Nine Open Crusader Helm'
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'Knights of the Nine Open Crusader Helm ESP' ]
+        condition: 'many("(Knights-OpenCrusaderHelm|EVE_Knights-OpenCrusaderHelm)\.esp")'
```

The regex can be simplified. It can stay where it is in the masterlist since the main mod's filename starts with a `K`.

```suggestion
  - name: '(EVE_)?Knights-OpenCrusaderHelm\.esp'
    url:
      - link: 'https://www.nexusmods.com/oblivion/mods/49319/'
        name: 'Knights of the Nine Open Crusader Helm'
    msg:
      - <<: *useOnlyOneX
        subs: [ 'Knights of the Nine Open Crusader Helm ESP' ]
        condition: 'many("(EVE_)?Knights-OpenCrusaderHelm\.esp")'
```